### PR TITLE
Default roles

### PR DESCRIPTION
**Pull request by a-belhadj**

### CODE REVIEW

File: profiles/default_rbac/default_roles.py [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        "permissions": [
+            "service_catalog.view_instance",
+            "service_catalog.view_request",
+            "service_catalog.view_requestmessage",
+            "service_catalog.view_supportmessage",
+            "service_catalog.view_support",
+
+        ]
+    },
+    "Operator": {
```
Comment (EliasBoulharts) [inline profiles/default_rbac/default_roles.py:51]: Name TBD

File: profiles/default_rbac/default_roles.py [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        "description": "Can request services and day 2 operations",
+        "permissions": [
+            "service_catalog.view_instance",
+            "service_catalog.view_request",
+            "service_catalog.view_requestmessage",
+            "service_catalog.view_supportmessage",
+            "service_catalog.view_support",
+
+            "profiles.consume_quota_scope",
```
Comment (EliasBoulharts) [inline profiles/default_rbac/default_roles.py:60]: Split to another role?

File: tests/test_service_catalog/base.py [hunk 3]
```diff
@@ -5,7 +5,7 @@
 from rest_framework.test import APITestCase
 
 from profiles.models import Organization, Scope, Role, GlobalPermission, Team
-from service_catalog.models import TowerServer, JobTemplate, Operation, Service, Portfolio
+from service_catalog.models import TowerServer, JobTemplate, Operation, Service, Portfolio, Instance
```
Comment (EliasBoulharts) [inline tests/test_service_catalog/base.py:8]: why do you import Instance?

Comment (a-belhadj) [reply tests/test_service_catalog/base.py]: removed

File: profiles/default_rbac/default_roles.py (hunk 1 above)
Comment (EliasBoulharts) [reply profiles/default_rbac/default_roles.py:51]: Instance operator

File: profiles/default_rbac/default_roles.py (hunk 2 above)
Comment (EliasBoulharts) [reply profiles/default_rbac/default_roles.py:60]: Catalog user

**Review by EliasBoulharts — APPROVED** [review]

**Pull request merged**